prac_06/guitar.py

```diff
@@ -0,0 +1,33 @@
+"""Guitar Class Program"""
+
+CURRENT_YEAR = 2024
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """Guitar class: name, year, cost."""
+
+    def __init__(self, name="", year=0, cost=0.0):
+        """Construct guitar object."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Return guitar object summary."""
+        return "{} ({}) : ${:,.2f}".format(self.name, self.year, self.cost)
```

Comment:
Down to personal preference but using f{} instead of format for improved readability

Reply:
Thanks for the feedback, I too prefer f-strings, but the question contained:
"__str__ - which uses {} string formatting..."
Which I interpreted as using .format, and it is worth practising other methods.